### ISSUE TITLE
QRNG chains are unsorted

### ISSUE THREAD

**Issue by dcroote** [api3dao/vitepress-docs#702]
https://docs.api3.org/reference/qrng/chains.html

How chains appear, by provider:
- ANU: by chain name
- Testnet: by chain ID, except ethereum-sepolia-testnet
- Quintessence: by chain name, except newly added names are unsorted at the bottom
- Quantum Blockchains: Same as Quintessence

We should have this standardized. I lean in favor of name, since a user may not have memorized the chain ID of the chain, but they will certainly know its name.

We can also then update the [Airnode contract](https://docs.api3.org/reference/airnode/latest/) sort scheme to match.

@wkande @vanshwassan @Ashar2shahid ?

**Comment by vanshwassan** [comment]
Yeah. sorting it with chain name makes more sense IMO. You can assign it to me, will make a PR for this.
